odalic/src/main/java/cz/cuni/mff/xrg/odalic/outputs/annotatedtable/DefaultResultToAnnotatedTableAdapter.java

```diff
@@ -57,24 +61,24 @@ public AnnotatedTable toAnnotatedTable(Result result, Input input, Configuration
     return new AnnotatedTable(input.identifier(), new TableSchema(columns));
   }
   
-  private TableColumn createOriginalColumn(String columnName) {
+  private TableColumn createOriginalColumn(String columnName, boolean chosenIsEmpty) {
     return new TableColumn(columnName, Arrays.asList(columnName),
-        "", "anyURI",
+        "", chosenIsEmpty ? "" : "anyURI",
         false, false,
-        bracketFormat(urlFormat(columnName)), "", "dcterms:title", "");
+        chosenIsEmpty ? "" : bracketFormat(urlFormat(columnName)), "", chosenIsEmpty ? "" : "dcterms:title", "");
   }
   
   private TableColumn createClassificationColumn(String columnName, String resource) {
     return new TableColumn(typeFormat(columnName), Arrays.asList(""),
         "", "",
         true, false,
-        bracketFormat(urlFormat(columnName)), "", "rdf:type", bracketFormat(resource));
+        bracketFormat(urlFormat(columnName)), "", "rdf:type", resource);
   }
   
-  private TableColumn createDisambiguationColumn(String columnName, boolean isSubjectColumn) {    
+  private TableColumn createDisambiguationColumn(String columnName) {
```

Comment:
I'm sorry, but I will have to leave fixing this to you after all. The use of empty strings is too frequent and you will need the model fixed anyway to continue the work on the conversion to RDF. Just use nulls when the attribute is not present and follow @tomas-knap instructions.

Avoid making multiple constructors or providing nulls to the one initializing all fields directly. Use the builder pattern instead to construct TableColumn instances with optionally present attributes.